raiden/api/python.py

```diff
@@ -648,34 +646,27 @@ def get_channel_events(
                 self.raiden.chain,
                 token_network_address,
                 channel.identifier,
-                from_block=from_block,
```

Comment:
Hmmm by doing this, we no longer are able to query events by block range from the API. Why would we want to remove this functionality?

Reply:
>  Why would we want to remove this functionality?

The PR is removing the block number, this filter is not possible anymore. the rationale is in the PR description.

To filter the api client will have to use limit+offset https://github.com/raiden-network/raiden/issues/2043